rust_dev_preview/cross_service/rest_ses/src/client.rs

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! Application logic aware wrappers around AWS SDK Clients.
+//!
+//! These wrappers add default ARNs for resources the application accesses, and adds those to common calls.
```

Comment:
In "adds those to common calls", does "those" refer to the ARNs or the resources?